src/components/FileUploader/FileUploader.js

```diff
@@ -18,6 +18,7 @@ export class FileUploaderButton extends Component {
     role: PropTypes.string,
     tabIndex: PropTypes.number,
     buttonKind: PropTypes.oneOf(['primary', 'secondary']),
+    accept: PropTypes.arrayOf(PropTypes.string),
```

Comment:
Are the potential string values bounded here? If so, might be helpful to use `PropTypes.oneOf` to make the prop type more strict, but understand if the list is wayyyy to large for that lol.